docs/index.rst

```diff
@@ -93,7 +93,8 @@ In addition to this documentation set, check out the following resources:
 - `Blog <https://neptune.ai/blog>`_: Provides in-depth articles about best practices and trends in machine learning.
 - `Neptune user community <https://spectrum.chat/neptune-community?tab=posts>`_: Meet other Neptune users and developers and start a discussion.
 - Presentations, talks, podcasts
-- Technical Support: Should you require further support, or have feature requests, reach out at contact@neptune.ai or click the chat icon in the bottom right corner of the Neptune UI.
+- Product hunt: a review helps other people find our product.
```

Comment:
The link to Product hunt profile is missing here.
https://www.producthunt.com/posts/neptune-ai

Reply:
That's what I mean when I said it didn't have enough "meat".
Adding it now.